### DISCUSSION TITLE
Create new template contracts

### DISCUSSION THREAD

**Discussion by guilhermelirio** [crewAIInc/crewAI#1087] [Q&A]
I've been using CrewAI for a while, but haven't used it for a few months. What do you think is the best approach to creating new contracts from .docx and .pdf templates?

**Comment by theCyberTech** [comment]
It think you will need a custom tool that takes a DOCX template and insert the required data, there are plenty of docx python modules out there

**Reply by guilhermelirio** [reply]
Thanks for the answer. Actually, I don't want you to fill in the data (because a prompt in OpenAI would solve the problem).

What I really want is for AI to generate new contract models from all the models it has.

**Reply by theCyberTech** [reply]
Can you explain what you mean by contract models
